packages/actor-abstract-bindings-hash/components/Actor/AbstractFilterHash.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-abstract-filter-hash/^1.0.0/components/context.jsonld",
```

Comment:
Wrong context.

Be sure to test with a full SPARQL query as well, because I assume this would not work at the moment.